common/common/src/main/java/io/helidon/build/common/FileUtils.java

```diff
@@ -747,6 +750,36 @@ public static Path zip(Path zip, Path directory) {
         }
     }
 
+    /**
+     * Unzip a zip file.
+     *
+     * @param zip       source file
+     * @param directory target directory
+     */
+    public static void unzip(Path zip, Path directory) {
+        try {
+            File destDir = directory.toFile();
+            if (!destDir.exists()) {
+                Files.createDirectory(destDir.toPath());
+            }
+            ZipInputStream zipIn = new ZipInputStream(new FileInputStream(zip.toString()));
```

Comment:
Use NIO instead, i.e construct a FileSystem for the zip and use `Files.list` or `Files.walk`.